### PR TITLE
Convert Sarif.FunctionalTests to Xunit

### PR DESCRIPTION
**Pull request by lgolding**
@michaelcfanning

### CODE REVIEW

File: src/Sarif.FunctionalTests/Sarif.FunctionalTests.csproj [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+  <Import Project="..\packages\xunit.runner.visualstudio.2.1.0\build\net20\xunit.runner.visualstudio.props" Condition="Exists('..\packages\xunit.runner.visualstudio.2.1.0\build\net20\xunit.runner.visualstudio.props')" />
   <PropertyGroup>
     <ProjectGuid>{195675E6-8FDF-4A87-90A0-242DB7F46B15}</ProjectGuid>
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis.Sarif</RootNamespace>
     <AssemblyName>Sarif.FunctionalTests</AssemblyName>
-    <ProjectTypeGuids>{3AC096D0-A1C2-E12C-1390-A8335801FDAB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
   </PropertyGroup>
```
Comment (lgolding) [inline src/Sarif.FunctionalTests/Sarif.FunctionalTests.csproj]: NuGet inserted these lines. Not sure why, but no reason to fight it.

File: src/Sarif.FunctionalTests/Sarif.FunctionalTests.csproj [hunk 2]
```diff
@@ -15,8 +17,13 @@
...
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., build.props))\build.props" />
   <ItemGroup>
-    <Reference Include="Microsoft.VisualStudio.QualityTools.UnitTestFramework, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
```
Comment (lgolding) [inline src/Sarif.FunctionalTests/Sarif.FunctionalTests.csproj]: No longer need MSTest assembly

File: src/Sarif.FunctionalTests/Sarif.FunctionalTests.csproj [hunk 3]
```diff
@@ -1,11 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\xunit.runner.visualstudio.2.1.0\build\net20\xunit.runner.visualstudio.props" Condition="Exists('..\packages\xunit.runner.visualstudio.2.1.0\build\net20\xunit.runner.visualstudio.props')" />
   <PropertyGroup>
     <ProjectGuid>{195675E6-8FDF-4A87-90A0-242DB7F46B15}</ProjectGuid>
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis.Sarif</RootNamespace>
     <AssemblyName>Sarif.FunctionalTests</AssemblyName>
-    <ProjectTypeGuids>{3AC096D0-A1C2-E12C-1390-A8335801FDAB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
```
Comment (lgolding) [inline src/Sarif.FunctionalTests/Sarif.FunctionalTests.csproj]: No longer need GUIDs marking this as an MSTest project. Xunit test projects are vanilla class libraries.

**Comment by michaelcfanning** [comment]
Code looks fine. I will swing by to take a look at how debuggable failures are now in VS.

**Comment by lgolding** [comment]
+1 over the shoulder from @michaelcfanning

**Pull request merged**